src/main/kotlin/frc/robot/autonomous/AutoConstants.kt

```diff
@@ -8,9 +8,33 @@ import edu.wpi.first.units.measure.Distance
 import edu.wpi.first.units.measure.LinearVelocity
 import edu.wpi.first.wpilibj.Filesystem
 import frc.robot.lib.Gains
+import frc.robot.lib.getPose2d
 import java.io.File
 import kotlinx.serialization.json.*
 
+private val Reef1: Pose2d = Pose2d(6.063, 3.980, Rotation2d.k180deg)
+private val Reef2: Pose2d = Pose2d(5.315, 5.366, Rotation2d.fromDegrees(-120.0))
+private val Reef3: Pose2d = Pose2d(3.769, 5.416, Rotation2d.fromDegrees(-60.0))
+private val Reef4: Pose2d = getPose2d(2.93, 4.030)
+private val Reef5: Pose2d = Pose2d(3.719, 2.644, Rotation2d.fromDegrees(60.0))
+private val Reef6: Pose2d = Pose2d(5.255, 2.624, Rotation2d.fromDegrees(120.0))
```

Comment:
We have a beautiful function that parses Choreo poses and even have a variable for that... ask @Emma03L. I don't see why you need to duplicate work already done by your teammate.